persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_5/generated/entities/generated_client.bal

```diff
@@ -4,20 +4,201 @@
-    isolated resource function delete medicalneed/[int needId]() returns MedicalNeed|persist:Error = external;
-
-    isolated resource function get medicalitem() returns stream<MedicalItem, persist:Error?> = external;
-    isolated resource function get medicalitem/[int itemId]() returns MedicalItem|persist:Error = external;
-    isolated resource function post medicalitem(MedicalItemInsert[] data) returns int[]|persist:Error = external;
-    isolated resource function put medicalitem/[int itemId](MedicalItem value) returns MedicalItem|persist:Error = external;
-    isolated resource function delete medicalitem/[int itemId]() returns MedicalItem|persist:Error = external;
+
+    private final mysql:Client dbClient;
+
+    private final map<persist:SQLClient> persistClients;
+
+    private final record {|persist:Metadata...;|} metadata = {
+        "medicalneed": {
+            entityName: "MedicalNeed",
+            tableName: `MedicalNeed`,
+            fieldMetadata: {
+                needId: {columnName: "needId", 'type: int},
+                itemId: {columnName: "itemId", 'type: int},
+                name: {columnName: "name", 'type: string},
+                beneficiaryId: {columnName: "beneficiaryId", 'type: int},
+                period: {columnName: "period", 'type: time:Civil},
+                urgency: {columnName: "urgency", 'type: string},
+                quantity: {columnName: "quantity", 'type: string}
+            },
+            keyFields: ["needId"]
+        },
+        "medicalitem": {
+            entityName: "MedicalItem",
+            tableName: `MedicalItem`,
+            fieldMetadata: {
+                itemId: {columnName: "itemId", 'type: int},
+                name: {columnName: "name", 'type: string},
+                'type: {columnName: "'type", 'type: string},
+                unit: {columnName: "unit", 'type: int}
+            },
+            keyFields: ["itemId"]
+        }
+    };
+
+    public function init() returns persist:Error? {
+        mysql:Client|error dbClient = new (host = host, user = user, password = password, database = database, port = port);
+        if dbClient is error {
+            return <persist:Error>error(dbClient.message());
+        }
+        self.dbClient = dbClient;
+        self.persistClients = {
+            medicalneed: check new (self.dbClient, self.metadata.get(MEDICAL_NEED)),
+            medicalitem: check new (self.dbClient, self.metadata.get(MEDICAL_ITEM))
+        };
+    }
+
+    isolated resource function get medicalneed() returns stream<MedicalNeed, persist:Error?> {
+        stream<record {}, sql:Error?>|persist:Error result = self.persistClients.get(MEDICAL_NEED).runReadQuery(MedicalNeed);
+        if result is persist:Error {
+            return new stream<MedicalNeed, persist:Error?>(new MedicalNeedStream((), result));
+        } else {
+            return new stream<MedicalNeed, persist:Error?>(new MedicalNeedStream(result));
+        }
+    }
```

Comment:
Shall we add a newline after every resource function?

Reply:
Added in [2ca4a7f](https://github.com/ballerina-platform/persist-tools/pull/108/commits/2ca4a7fd574065c9e56b86a8da6e78877e9cca5e)

Reply:
Updated in [2ca4a7f](https://github.com/ballerina-platform/persist-tools/pull/108/commits/2ca4a7fd574065c9e56b86a8da6e78877e9cca5e)